tests/wamr-test-suites/test_wamr.sh

```diff
@@ -65,7 +67,7 @@ QEMU_FIRMWARE=""
 # prod/testsuite-all branch
 WASI_TESTSUITE_COMMIT="ee807fc551978490bf1c277059aabfa1e589a6c2"
 
-while getopts ":s:cabgvt:m:MCpSXxwPGQF:j:" opt
+while getopts ":s:cabgvt:m:MCpSXxwPGQFf:j:" opt
```

Comment:
Could we get the `WASI_TEST_FILTER` from the environment variable, like the `TSAN_OPTIONS` in https://github.com/bytecodealliance/wasm-micro-runtime/pull/2679? A concern is that it is a not so popular config, we had better not use one character for the option, we may add more wasm features in the future, and may need to more characters.

Reply:
Sure, will change to use an environment variable.